#Header header    # ROS Message headaer

uint8 target      # Target
int32 error0      # Axis 0 error
int32 state0      # Axis 0 state

int32 error1      # Axis 1 error
int32 state1      # Axis 1 state

